content/departments/product-engineering/engineering/cloud/delivery/on-call.md

```diff
@@ -0,0 +1,126 @@
+# Delivery Team On-Call Rotation
+
+The Delivery Team maintains an on-call rotation where an engineer is dedicated to providing support to our
+internal teams and managed instances for a term of 1 week.
+
+The on-call schedule is captured in OpsGenie. The on-call rotation consists of an Engineer On-Call (EOC), a Secondary Engineer On-Call (SEOC), and a technical manager (TM).
+
+On-Call duties transfer automatically Mondays at 10am Central Time according to the schedule within OpsGenie.
+
+## Response Times and SLAs
+
+The Delivery team aims to provide timely responses that are proportional to the capacity of the team.
+The Engineer On-Call (EOC) is responsible for a 24 hour x 7 period shift, guarenteeing responses during their
+working hours (generally 9am - 5pm in their timezone).
```

Comment:
I don't love this definition. I'd like to be able to figure out how this could be more async-friendly.

Reply:
Yeah, I think there's a distinction here between on-call (being paged by Opsgenie, guaranteed response) and support responses (some kind of response within 24 hours).

Reply:
I reworked this section a bit. Feel free to re-open this comment if it's still not reading correctly :-)